lib/core/src/Cardano/Wallet/DB/Sqlite/Types.hs

```diff
@@ -225,9 +226,16 @@ instance PathPiece BlockId where
 instance PersistFieldSql SlotId where
     sqlType _ = sqlType (Proxy @Word64)
 
+-- | As a short-to-medium term solution of persisting 'SlotId', we use
+-- 'flatSlot' with an artificial epochLength. I.e. /not the same epochLength as
+-- the blockchain/. This is just for the sake of storing the 64 bit epoch and
+-- the 16 bit slot inside a single 64-bit field.
+artificialEpochLength :: SlotsPerEpoch
+artificialEpochLength = SlotsPerEpoch $ fromIntegral (maxBound :: Word16)
```

Comment:
Maybe worth mentioning that this artificial epoch length **must** be greater than the actual one. Otherwise, we may end up with a wrong result when deserializing.

Reply:
As in making sure no-one changes it from `maxBound :: Word16` to say `50`?

Because the slotting arithmetic should already break down with an epoch length greater than `Word16` because `slotNumber :: Word16` should overflow.